docs/evaluation/index.md

```diff
@@ -6,6 +6,7 @@ We support many popular benchmarks and it's easy to add new in the future. The f
 - [**Math (formal language)**](./formal-math.md): e.g. [minif2f](./formal-math.md#minif2f), [proofnet](./formal-math.md#proofnet), [putnam-bench](./formal-math.md#putnam-bench)
 - [**Code**](./code.md): e.g. [swe-bench](./code.md#swe-bench), [livecodebench](./code.md#livecodebench)
 - [**Scientific knowledge**](./scientific-knowledge.md): e.g., [hle](./scientific-knowledge.md#hle), [scicode](./scientific-knowledge.md#scicode), [gpqa](./scientific-knowledge.md#gpqa)
+- [**Speech and Audio Understanding**](./speech-audio.md): e.g. [mmau-pro](./speech-audio.md#mmau-pro)
```

Comment:
Please also add to https://github.com/NVIDIA-NeMo/Skills/blob/main/README.md?plain=1#L17